internal/blobcache/blobcache.go

```diff
@@ -11,29 +11,29 @@ import (
 
 // Crude estimate of the overhead per blob: a SHA-256, a linked list node
 // and some pointers. See comment in blobCache.add.
-const cacheOverhead = len(restic.ID{}) + 64
+const CacheOverhead = len(restic.ID{}) + 64
 
 // A blobCache is a fixed-size cache of blob contents.
 // It is safe for concurrent access.
-type blobCache struct {
+type BlobCache struct {
```

Comment:
This can be renamed Cache.

Reply:
Hm, I'm undecided because I don't speak golang fluently enough to have an opinion here. I'd love a second opinion here, pleaase

Reply:
The Go convention is to [avoid stutter](https://sben.dev/stutter/).

Reply:
Ah, that makes perfect sense. I'll update the code. Thank you for the link